### PR TITLE
Create and interpret a model

### LINKED ISSUE #6: Creating a binary logistic regression model

### LINKED ISSUE #7: Relevant explanations of the model and analysis of the conclusions

### PR DESCRIPTION
**Pull request by Gu-YuxuanLi**
Resolve #6 
Resolve #7

### CODE REVIEW

**Review by Yunfei-Ch — APPROVED** [review]

**Review by shuyin1013 — APPROVED** [review]

File: Group_13_Analysis.qmd [hunk 1]
```diff
@@ -152,3 +152,47 @@ $$characteristics_{\mbox{i}} =
...
+## Creating model
+
+```{r}
+#formula
+formula1 <- as.formula(paste("Qualityclass_binary ~", paste(names(data1)[1:4], collapse = "+")))
+
+# Fit model for the Qualityclass_binary
+logistic_model1 <- glm(formula1, data = data1, family = binomial)
+
+summary(logistic_model1)
```
Comment (Zoey-2024) [inline Group_13_Analysis.qmd:164]: some variables are insignificant

File: Group_13_Analysis.qmd [hunk 2]
```diff
@@ -152,3 +152,47 @@ $$characteristics_{\mbox{i}} =
...
+# Filter variables using stepwise regression
+stepwise_model <- step(logistic_model1)
+
+summary(stepwise_model)
```
Comment (Zoey-2024) [inline Group_13_Analysis.qmd:168]: the model fit well

File: Group_13_Analysis.qmd [hunk 3]
```diff
@@ -152,3 +152,47 @@ $$characteristics_{\mbox{i}} =
...
+coefficients_table <- model_summary$coefficients
+
+write.csv(coefficients_table, "logistic_model_summary.csv", row.names = TRUE)
+
+```
+
+According to our analysis, we can get the model：
+$$ {Qualityclass\_binary}_i = \beta_0 + \beta_1 \cdot{characteristics}_i + \beta_2  \cdot{altitude\_mean\_meters}_i + \varepsilon_i $$
+
+where
```
Comment (Zoey-2024) [inline Group_13_Analysis.qmd:194]: good job!

**Review by Zoey-2024 — APPROVED** [review]

**Review by 1162259403WXR — APPROVED** [review]
It is reasonable to use a binary logistic regression model.

**Comment by shuyin1013** [comment]
Good.

**Pull request merged**